lib/prefix.h

```diff
@@ -174,6 +184,13 @@ struct prefix_ptr {
 	uintptr_t prefix __attribute__((aligned(8)));
 };
 
+/* Prefix for a Flowspec entry */
+struct prefix_fs {
+	u_char family;
+	u_char prefixlen; /* unused */
```

Comment:
flowspec_prefix has the actual prefix length?  This just seems broken to me as that it breaks the prefix model.

Reply:
The prefix model uses a prefix length that is a u_char and that is expressed in bits.
From RFC5575, the FS NLI coontains a length field:
-  a 1- or 2-octet NLRI length field followed by a variable-length NLRI
- The NLRI length is expressed in octets.
the value can be over 255 bytes, which is why I propose to create a new prefix_fs, and not chaing the prefixlen field from 8 bits to 16 bits